tutorials/admin/getting-started/installation.md

```diff
@@ -1 +1,237 @@
+just copying `env.local.example` to `env.local`
+```
+
+### Configure `env.local`
+
+The `env.local` file contains environment variables that are used when deploying the Marble software.
+
+Instructions on which variables should be set in this file can be found in `env.local.example`. Please refer to this
+file which will instruct you:
+
+- which variables *must* be set for all deployments
+- which variables can be modified to customize your deployment
+- how to enable additional components and which variables to set to customize these components
+
+(deploy-marble)=
+## Deploying Marble
+
+The stack can be started, stopped and inspected using the `pavics-compose.sh` script which takes the same arguments as
+the [`docker compose`](https://docs.docker.com/compose/) commands.
+
+The `pavics-compose.sh` script will read the `env.local` configuration file, create a `docker compose` deployment file
+based on this configuration and then execute the subcommand based on that file. 
+
+For example, to start the Marble stack:
+
+```shell
+cd birdhouse-deploy/birdhouse
+./pavics-compose.sh up --detach
+```
+
+This will call the [`docker compose up --detach`](https://docs.docker.com/engine/reference/commandline/compose_up/)
+which will start up all the docker containers required to run the Marble stack.
+
+Similarly, to stop and bring down the stack use:
+
+```shell
+cd birdhouse-deploy/birdhouse
+./pavics-compose.sh down
+```
+
+## Check that everything is working
+
+Once you have started the Marble stack with the `./pavics-compose.sh up --detach` command, check that everything is
+working properly by opening your webpage in a browser:
+
+- Navigate to the domain name [that you set up](fqdn)
+- Ensure that the page loads as expected
+  - Note that the page that loads should be the one set by the `PROXY_ROOT_LOCATION` variable in `env.local`
+- Ensure that the connection is secure (uses the https protocol with a valid certificate)
+  - If not: ensure that your SSL certificate is up-to-date and that `SSL_CERTIFICATE` variable in `env.local` is
+    pointing to the correct file (see the birdhouse documentation for details)
+
+## Next Steps
+
+Now that you know the basics of how to configure and deploy the Marble stack you can select which optional components
+you would like to add to your stack:
+
+- [Monitoring and Logging components](../monitoring/monitoring.md)
+- [User-facing Services](../services/services.md)
+
+Once you've done that, you will likely want to add your new node to the Marble network:
+
+- [Network](../network/network.md)
+
+If you want to host data on your node:
+
+- [Data hosting](../data/data.md)
+
+And finally you'll want to start adding users to your node:
+
+- [User management](../users/users.md)
+
+## Example installation steps
+
+Here is an example of how to install Marble on a server running Ubuntu 20.04, using mymarble.com as a FQDN, and where we
```

Comment:
mymarble.com is getting hyperlinked, but it shouldn't. To prevent that you could do: <span>mymarble</span>.com.